manifests/istio-control/istio-discovery/templates/clusterrolebinding.yaml

```diff
@@ -12,23 +12,23 @@ roleRef:
   name: istio-pilot-{{ .Release.Namespace }}
 subjects:
   - kind: ServiceAccount
-    name: istio-pilot-service-account
+    name: istiod-service-account
```

Comment:
Do we still need the istio-pilot clusterrolebinding if everything is covered by the istiod equivalents? I think the only reason for keeping this around is to avoid breaking existing 1.4 installs and to support 1.5 with non-istiod deployments (if we even test/support that).

Reply:
We can drop the non-istiod deployments in master. However when this is backported extra care needs to be taken to keep the pilot and istiod versions in parallel, right?

Reply:
We can definitely drop it in master. I suspect we don't need both in 1.5, but we do support a non-istiod mode (it still called istiod, but it doesn't do injection/CA/validation) in 1.5